src/components/Card/Card.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { IAnime } from "../../types/interfaces/IAnime";
+import styles from "./card.module.scss"
+
+interface CardProps {
+    anime: IAnime
+}
+
+export function Card({ anime }: CardProps) {
+    const navigate = useNavigate();
+
+    const handleClick = (): void => {
```

Comment:
```suggestion
    const handleClick = () => {
```
типизация здесь излишняя

Reply:
исправил